introduction.md

```diff
@@ -0,0 +1 @@
+Hi, I'm Jeremy, an aspiring developer from Central Texas. My favorite Code School path has been .NET.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.